.github/workflows/ci.yml

```diff
@@ -54,4 +54,4 @@ jobs:
           pytest --cov=removestar . -vv
 
       - name: Upload coverage report
-        uses: codecov/codecov-action@v3.1.4
+        uses: codecov/codecov-action@v5.0.7
```

Comment:
@asmeurer this would require adding the `CODECOV_TOKEN` to repository secrets. The uploading coverage report job fails silently in this PR. Could you please add that whenever you are free? Thanks! 🙂

Reply:
Agreed.  There was a breaking change at https://github.com/codecov/codecov-action?tab=readme-ov-file#breaking-changes that required this.